### ISSUE TITLE
Code problem

### ISSUE THREAD

**Issue by miss-rain** [CDTrans/CDTrans#3]
in **processor_uda.py**

<img width="377" alt="1" src="https://user-images.githubusercontent.com/17979331/148628961-18df75b2-5248-468c-bc52-2b0ee4a50a79.png">

line 236: train_loader, train_loader1, and  train_loader2,
 **I don't konw train_loder1&2 means.**

**Comment by miss-rain** [comment]
in **processor_uda.py**

<img width="918" alt="2" src="https://user-images.githubusercontent.com/17979331/148629073-e3b71997-39a8-4a1f-94e6-62cdeae3dadd.png">
<img width="779" alt="3" src="https://user-images.githubusercontent.com/17979331/148629078-8d517de5-c1f5-40f3-b580-06519f7fe4d4.png">

line 318, for train :
**i don't understand the means of parameters:'imgs, vid, target_cam, target_view, file_name, idx'**

also in line 472, for inference:
**I don't understand 'img, vid, camid, camids, target_view, _'**

Thanks again.

**Comment by CDTrans** [comment]
> 



> in **processor_uda.py**
> 
> <img alt="1" width="377" src="https://user-images.githubusercontent.com/17979331/148628961-18df75b2-5248-468c-bc52-2b0ee4a50a79.png">
> 
> line 236: train_loader, train_loader1, and train_loader2, **I don't konw train_loder1&2 means.**

The train_loader1 and train_loader2 are the source dataloader and target dataloader respectively, which are defined in the make_dataloader function of the dataset/make_dataloader.py file

**Comment by CDTrans** [comment]
The parameters of imgs, vid, file_name, and idx represent the image tensor, label, image file name, and image index value, respectively.  

The parameters of the target_cam (camid, camids), target_view are inherited from the TransReID project and is a parameter potentially used by ReID, but are not used in this project now.

**Comment by gliehu** [comment]
1. in vit_pytorch_uda.py,    Line329,     Block funtion ERROR.     Should it replaced with Block_3_branches?
![error](https://user-images.githubusercontent.com/40121528/149529739-74a8db2e-1bb2-4c50-8ba8-f7a49eebec49.png)
2. in scripts/pretrain/**officehome**/*.sh files: DATASETS.ROOT_TRAIN_DIR './data/office31/Art.txt'    DATASETS.ROOT_TEST_DIR './data/office31/Product.txt'   why include the DIR of **office31** dataset?

**Comment by CDTrans** [comment]
> 1. in vit_pytorch_uda.py,    Line329,     Block funtion ERROR.     Should it replaced with Block_3_branches?
>    ![error](https://user-images.githubusercontent.com/40121528/149529739-74a8db2e-1bb2-4c50-8ba8-f7a49eebec49.png)
> 2. in scripts/pretrain/**officehome**/*.sh files: DATASETS.ROOT_TRAIN_DIR './data/office31/Art.txt'    DATASETS.ROOT_TEST_DIR './data/office31/Product.txt'   why include the DIR of **office31** dataset?

Thanks for your correction. 

I removed the VisionTransformer and VisionTransformer_mask classes of vit_pytorch_uda.py as it is not used in the project. The VisionTransformer and VisionTransformer_mask classes are correctly written in vit_pytorch.py if it is useful to you.

I also updated the scripts/pretrain/officehome/*.sh file. The configuration dataset in the file should be 'OfficeHomeDataset' instead of 'office31'.

Thanks.

**Comment by CDTrans** [comment]
I haven't heard from you for a long time, I will now close this issue. If you have new questions, you can reopen the issue。
